src/modules/numbers/numbers.service.ts

```diff
@@ -0,0 +1,58 @@
+import { Injectable, NotFoundException} from '@nestjs/common';
+import { CreateNumberDto } from './dto/create-number.dto';
+import { InjectRepository } from '@nestjs/typeorm';
+import { Repository } from 'typeorm';
+import { UkefNumber as Number } from './entities/number.entity';
+
+@Injectable()
+export class NumbersService {
+
+  constructor(
+    @InjectRepository(Number, "mssql-number-generator")
+      private readonly numberRepository: Repository<Number>,
+    ) {}
+
+  async create(createNumbers: CreateNumberDto[]): Promise<Number[]> {
+    let results = []
```

Comment:
in the interest of keeping everything consistent, can we add semicolons here and to the variable below?

Reply:
can we also conver it to `const` - the variable is not being re-assigned

Reply:
Eslint added semicolon and converted variable types.